sections/main-product.liquid

```diff
@@ -113,7 +113,7 @@
                   </div>
                   {% render 'icon-caret' %}
                 </summary>
-                <div class="accordion__content rte">
+                <div class="accordion__content rte" id="ProductAccordion-{{ block.id }}-{{ section.id }}">
                   {{ block.settings.content }}
                   {{ block.settings.page.content }}
                 </div>
```

Comment:
I wonder if we should do the same approach for the share button and remove the duplicate work being done in `share.js` 🤔

Reply:
Looks like it would mean removing an event listener and the `addAccessibilityAttributes` function.